### PR TITLE
Feature/spectron-modify-action

### PR DESCRIPTION
**Pull request by Jess-Jian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "electron-store": "^6.0.0",
     "electron-updater": "^4.2.0",
     "eventemitter3": "^4.0.7",
-    "font-scanner": "dkdkwizard/font-scanner",
+    "font-scanner": "^0.1.2",
```
Comment (dkdkwizard) [inline package.json:30]: The font scanner has some issue, so I fork it and modify some places.

File: test/features/editor/modify-action-test/array-test.js [hunk 2]
```diff
@@ -1,417 +1,274 @@
...
-    const svg_11_text = await app.client.$('#svg_11');
-    expect(await svg_11_text.getAttribute('y')).toEqual(await svg_2_text.getAttribute('y'));
-    expect((await svg_11_text.getAttribute('x')-await svg_2_text.getAttribute('x'))).toEqual(1000);
-
-    const svg_15_text = await app.client.$('#svg_15');
-    expect((await svg_15_text.getAttribute('x')-await svg_2_text.getAttribute('x'))).toEqual(1000);
-    expect((await svg_15_text.getAttribute('y')-await svg_2_text.getAttribute('y'))).toEqual(1000);
+  };
+
+  async function checkSvg(number1, number2, number3, number4, number5, number6, number7, number8) {
```
Comment (dkdkwizard) [inline test/features/editor/modify-action-test/array-test.js:260]: You can write something like:
```
checkSvgIdExists(ids) {
  const promises = ids.map((id) => checkExist(`#svg_${id}`));
  await Promise.all(promises);
}
```

File: yarn.lock [hunk 3]
```diff
@@ -6140,9 +6140,10 @@ follow-redirects@^1.10.0:
   resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.14.0.tgz#f5d260f95c5f8c105894491feee5dc8993b402fe"
   integrity sha512-0vRwd7RKQBTt+mgu87mtYeofLFZpTas2S9zY+jIeuLJMNvudIgF52nr19q40HOwH5RrhWIPuj9puybzSJiRrVg==
 
-font-scanner@dkdkwizard/font-scanner:
+font-scanner@^0.1.2:
```
Comment (dkdkwizard) [inline yarn.lock:6143]: Same as package.json

File: package.json (hunk 1 above)
Comment (Jess-Jian) [reply package.json]: updated

File: test/features/editor/modify-action-test/array-test.js (hunk 2 above)
Comment (Jess-Jian) [reply test/features/editor/modify-action-test/array-test.js]: updated

File: yarn.lock (hunk 3 above)
Comment (Jess-Jian) [reply yarn.lock]: updated

**Review by dkdkwizard — APPROVED** [review]

**Pull request merged**